src/components/dailyNotes/NotePreviewModal.tsx

```
@@ -1,238 +1,468 @@
+export const NotePreviewModal: React.FC<NotePreviewModalProps> = ({ note, onClose, onEdit, onUpdateNote, refetchNotes }) => {
   const modalRef = useRef<HTMLDivElement>(null);
+  const { user } = useAuth();
+  const [isGenerating, setIsGenerating] = useState(false);
+  const [showPaywall, setShowPaywall] = useState(false);
+  const [showFullscreen, setShowFullscreen] = useState(false);
+  const [zoomLevel, setZoomLevel] = useState(3);
+  const [panOffset, setPanOffset] = useState({ x: 0, y: 0 });
+  const [isDragging, setIsDragging] = useState(false);
+  const [dragStart, setDragStart] = useState({ x: 0, y: 0 });
+
+  const isProUser = (user as any)?.subscription_plan === 'pro';
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)